jvm-libs/generic/json-rpc/src/main/kotlin/net/consensys/linea/jsonrpc/JsonRpcMessageProcessor.kt

```diff
@@ -174,7 +181,13 @@ class JsonRpcMessageProcessor(
     return SimpleTimerCapture<String>(meterRegistry, "jsonrpc.serialization.response")
       .setDescription("Time of json response serialization")
       .setTag("method", requestContext.method)
-      .captureTime { Json.encode(requestContext.result.merge()) }
+      .captureTime {
```

Comment:
I don't really understand this change. Can you please explain the difference?

Reply:
Without this change we cannot encode numbers e.g Int/ULong/etc without affecting the JSON-RPC envelope. 

The goal is to support custom objectMapper to the jsonrpc.**result**.

Reply:
I see, so with this you use non-default object mapper with Kotlin support, is it correct?

Reply:
I mean, it is still backwards compatible